README.md

```diff
@@ -2,17 +2,49 @@
 
 > Official GitHub Platform for GitHub Apps in Node.js
 
-### GitHub Apps
+## GitHub Apps
 
-The difference to using `@octokit/app` standalone is that the current
-`client` instance will be passed as additional context property passed to
-the event handlers. The `client` instance will have default configuration set
-for the respective event
+`@octokit/app.js` allows users to authenticate as GitHub Apps, Listen on
+Webhooks, and respond to them using the GitHub API.
+
+### Authenticating (AKA Getting your Acess Token)
+
+In order to take action as a GitHub App, you need to create an Authentication
+Token. In order to do this you need to generate a Private Key and use it to
+sign a JSON Web Token (jwt) and encode it. You can then use that jwt in order
+to request an Acess Token.
+
+```js
+const auth = await appAuth({id, privateKey})
+const client = requestToken(auth)
+```
+
+### Listening on Webhooks
+
+GitHub Apps give you the ability to listen for webhook events that happen on
+GitHub and take action. The `client` also comes with a built in logger.
```

Comment:
👍 I thought about that as well, but maybe we should pass some sort of low level logging in case people wanna customize that?

For example in probot we use some tool for different logging levels (ie debug, trace, info, etc.)

Reply:
I think ideally `@octokit/app` would be so minimal that it wouldn’t need any internal logging, instead people using it directly could add their own logging around it?

Ideally I'd want people to be able to use their own logging libraries